remove_unwanted_files.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
Given that this is messing with sensitive data, I would suggest explicitly setting the PATH to avoid any sort of spoofage, intentional or otherwise:

export PATH=/bin:/usr/bin

Reply:
Do we need to capture current path / reset after the script so we don't accidentally blow away the previous system path?

Reply:
@e-gris Can you explain what could happen without PATH set like this?

Reply:
If I run,

$ env PATH=/home/haxor/bin remove_unwanted_files.sh

Under these conditions, ls is /home/haxor/bin/ls, grep is /home/haxor/bin/grep. They can do whatever they want.

By explicitly setting PATH in the script, ls and grep are (practically) guaranteed to do what you expect.

Reply:
@e-gris cool cool, thanks.

Would it be better to explicitly path ls and grep? I'm a monster who has `grep` aliased to `ag` so setting the path just makes it fail on my machine.

Reply:
@underscorgan  This gets complicated and has no easy solution. There are arguments for both. In this case, I generally suggest setting PATH.

To the best of my knowledge your grep alias wouldn't affect the running of a script since aliases are parsed only by the shell's command-line reader. I would have to see your use case to see if I'm misunderstanding something.

Reply:
@e-gris yeah, fair. I had just tested it in my shell, looks like it's okay in a script. My bash seems to be getting rusty...